### DISCUSSION TITLE
b.getLength is not a function. (In 'b.getLength()', '.b.getLength' is undefinded)

### DISCUSSION THREAD

**Discussion by hannerr** [diegoazh/gmap-vue#283] [Q&A]
Hi,
I've been getting this error loading a map, but not all the time. Sometimes it works fine for a while, sometimes not.
The markers and infowindow load fine...
I already had this problem with vue2-google-maps so i switched to gmap-vue - thanks for the much better Documentation btw ;-)
Any ideas on this topic?
I read about not calling b.getLength() like this but like so: getLength(b) - but this would imply changing the modules files, which i don't think is a good idea...
Thanks, hanna

**Comment by diegoazh** [comment]
Maintenance of this plugin for Vue v2 is stopped, please migrate to [`@gmap-vue/v3`](https://www.npmjs.com/package/@gmap-vue/v3?activeTab=readme)
